presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -119,6 +120,15 @@ public DbResourceGroupConfigurationManager(ClusterMemoryPoolManager memoryPoolMa
         if (lastRefresh.get() == 0) {
             throw new PrestoException(CONFIGURATION_UNAVAILABLE, "Root groups cannot be fetched from database");
         }
+
+        if (System.nanoTime() - lastRefresh.get() > maxRefreshInterval.toMillis() * 1_000_000) {
```

Comment:
The logic using `succinctNanos` is easier to read.

Reply:
@rongrong : But the underlying logic looks quite involved. Not sure if we want to call it per each second...

Reply:
Is it involved enough to cause coordinator perf issues? If not I won't care. If yes then change `maxRefreshInterval` to `maxRefreshIntervalNanos` instead and initialize it that way.

Reply:
Use `MILLISECONDS.toNanos(1)` instead of `1_000_000` to help readability :)